### PR TITLE
Menu: move stats into subscription management

### PR DESCRIPTION
**Pull request by math-GH**
stats are only (!) available via the drop down menu

grafik

subscription management does not have any connection to the stats
grafik

the stats were "alone":
grafik

Changes proposed in this pull request:
- stats are now available in the subscription management
- stats menu entry is dropped

stats are now available within the subscription management
grafik

the entry in the dropdown menu is dropped
grafik

Why does it make sense to change?
1. the stats were before only represented with 1 menu entry
2. the stats are part of category and feed management
3. the "idle feeds" stat manages (f.e. deletes) idleing feeds

IMHO it makes much more sense to combine the subscription management and the stats.

How to test the feature manually:
go to subsrciption management and walk around

Pull request checklist:

- [x] clear commit messages
- [x] code manually tested

### CODE REVIEW

File: app/layout/aside_subscription.phtml [hunk 1]
```diff
@@ -21,5 +21,17 @@
 		<li class="item<?= Minz_Request::controllerName() === 'subscription' && Minz_Request::actionName() === 'bookmarklet' ? ' active' : '' ?>">
 			<a href="<?= _url('subscription', 'bookmarklet') ?>"><?= _t('sub.menu.subscription_tools') ?></a>
 		</li>
+		<li class="nav-header"><?= _t('admin.stats') ?></li>
+		<li class="item<?= Minz_Request::controllerName() == 'stats' && Minz_Request::actionName() == 'index' ? ' active' : '' ?>">
+			<a href="<?= _url('stats', 'index') ?>"><?= _t('admin.stats.menu.main') ?></a>
```
[the remark this answers is not in the document]
Comment (Alkarex) [reply app/layout/aside_subscription.phtml]: Indeed, that would make sense

[the remark this answers is not in the document]
Comment (math-GH) [reply app/layout/aside_subscription.phtml]: What array would you suggest?
admin.stats.menu(.main) change to sub.menu.stats(.main)?

sub.menu:
grafik

[the remark this answers is not in the document]
Comment (Alkarex) [reply app/layout/aside_subscription.phtml]: > What array would you suggest?
> admin.stats.menu(.main) change to sub.menu.stats(.main)?
> 

Yes, I think so, except if @aledeg has a better suggestion

[the remark this answers is not in the document]
Comment (aledeg) [reply app/layout/aside_subscription.phtml]: I think that makes sense

[the remark this answers is not in the document]
Comment (Alkarex) [reply app/layout/aside_subscription.phtml]: Does that sound good @mathContao ?

**Comment by math-GH** [comment]
fixed - ready for review

**Review by Alkarex — APPROVED** [review]

**Pull request merged**